lib/rexml/parsers/baseparser.rb

```diff
@@ -503,11 +509,10 @@ def normalize( input, entities=nil, entity_filter=nil )
       end
 
       # Unescapes all possible entities
-      def unnormalize( string, entities=nil, filter=nil )
-        rv = string.gsub( /\r\n?/, "\n" )
+      def unnormalize( rv, entities=nil, filter=nil )
```

Comment:
Could you avoid using `rv` name for parameter name?
`rv` is `return value`. It's not suitable for parameter name.

Reply:
OK
I see.